Notification email 3

gustavolimav opened the pull request.

Commit:
LPS-153050 Border bug in Dialect active tabs

jkappler committed:
LPS-147440 Take into account portal properties for definition of a page (name and friendly URL for example)

jkappler committed:
LPS-147440 Add support for layoutTemplateIds in site initializer

jkappler committed:
LPS-147440 Layout sets can also contain values that come from properties, so handle this

jkappler committed:
LPS-147440 Take into account the priority for layouts in site initializer extender

jkappler committed:
LPS-147440 Make release info available to the pages, in case the users want to work with them

jkappler committed:
LPS-147440 Create welcome site initializer based on existing one that is not using the new site initializer framework

jkappler committed:
LPS-147440 Remove site initializer and reference the new one

claraizquierdo committed:
LPS-152604 Add clear button to TreeFilter

jkappler committed:
LPS-147440 Site initializer extender reads PrincipalThreadLocal and ServiceContextThreadLocal, but we are starting the portal and it's not available, so populate it

jkappler committed:
LPS-147440 Cannot use GetterUtil here, since it doesn't allow StringPool.BLANK as second parameter, so us classic if instead

jkappler committed:
LPS-147440 Update tests

evanilson-santana committed:
LPS-153520 - Add CanViewFieldsLabelByDefault Test

evanilson-santana committed:
LPS-153521 - Add CannotEditDefaultFields Test

evanilson-santana committed:
LPS-153522 - Add CannotDeleteDefaultFields Test

evanilson-santana committed:
LPS-153523 - Add CanViewNameAndTypeAsMandatoryFields Test

qz1010 committed:
LPS-154006 Add logic for check group together for DB process Methods

qz1010 committed:
LPS-154006 Add testcase

qz1010 committed:
LPS-154006 Auto SF

qz1010 committed:
LPS-154006 Fix testcase

claraizquierdo committed:
LPS-152604 Adjust styles

claraizquierdo committed:
LPS-152604 Add tests for Clear button

bryceosterhaus committed:
LPS-153649 - Use onClick callback instead of href='javascript:'

jkappler committed:
LPS-147440 We needed references even if we are not using it, they have to be available for site initializer

Commit:
LPS-152059 Add CanAddMetadataColumnsToDefaultSort test

Commit:
LPS-152064 Add CanSearchWithEnterKey test

Commit:
LPS-152060 Add CanSortColumnEntries test

Commit:
LPS-152058 Add CanViewEntriesWithDefaultSort test

Commit:
LPS-152059 Small fixes

Commit:
LPS-153133 Add priority variable in curl

Commit:
LPS-153133 Add StructuredContentDraftContainsSetPriorityValue testcase

Commit:
LPS-153133 SF

evanilson-santana committed:
LPS-153524 - Add CanCreateActionOnSystemAccount Test

Commit:
LPS-151362 Added segmentation disabled warning message to the experience selector

bryceosterhaus committed:
LPS-153649 - Run baseline

Commit:
LPS-151362 buildLang

claraizquierdo committed:
LPS-152604 Improvements in Clear All link styles

claraizquierdo committed:
LPS-152604 Fix tests

Commit:
LPS-153050 - Apply fix consistently to all tab types and avoid changing border-width in an active state.

Commit:
LPS-154284 gw buildService

qz1010 committed:
LPS-154006 Move logic to JavaUpgradeEmptyLinesCheck.java

qz1010 committed:
LPS-154006 Auto SF

javalosjr96 committed:
LPS-113360 throw exception instead of warning

evanilson-santana committed:
LPS-153526 - Add CanViewTypeOnFieldEntry Test

evanilson-santana committed:
LPS-154224 - Delete CannotDeletePublishedObjectViaApi test

Kvale1733 committed:
COMMERCE-8975 Add macro to get specific product id

Kvale1733 committed:
COMMERCE-8975 Add json macro to get cpDefinitionId

Kvale1733 committed:
COMMERCE-8975 Add json macro to update product

Kvale1733 committed:
COMMERCE-8975 Add json macro to add diagram pin

Kvale1733 committed:
COMMERCE-8975 Add mapped product search entry result path

Commit:
LPS-154071 Add finder and remote service to queueEntry

Commit:
LPS-154071 Add update, send and resend methods in localService

Commit:
LPS-154071 Add resend and delete in queueEntry Service

Commit:
LPS-154071 Add DTO and Endpoints to openApi.yaml

Commit:
LPS-154071 Implement queueEntry resource

Commit:
LPS-154071 Add notificationHelper

Commit:
LPS-154071 Add tests

Commit:
LPS-154071 Add NotificationType and ObjectDefinitionNotificationType

Commit:
LPS-153133 Update task title

Kvale1733 committed:
COMMERCE-8534 Modify createNewOrderFromPendingOrders macro to select order type

Commit:
LPS-154071 buildService

Commit:
LPS-154071 buildService

Commit:
LPS-154071 buildService

Commit:
LPS-154071 buildRest

evanilson-santana committed:
LPS-153527 - Add CanViewNameOnFieldEntry Test

evanilson-santana committed:
LPS-153528 - Add CanViewDescriptionOnFieldEntry Test

ling-alan-huang committed:
LPS-154006 SF, rename

ling-alan-huang committed:
LPS-154006 Extending from BaseFileCheck is enough

ling-alan-huang committed:
LPS-154006 SF, simplify regex

ling-alan-huang committed:
LPS-154006 SF, rename

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

ling-alan-huang committed:
LPS-154006 SF

Commit:
LPS-154071 Add NotificationTypeRegistry and *impl

Commit:
LPS-154071 Add ObjectDefinitionNotificationType

Commit:
LPS-154071 Edit NotificationHelper and *impl to accept diferent types of notifications

Commit:
LPS-154071 Rename variables

Commit:
LPS-154071 buildService

Commit:
LPS-154071 Add finder By sentDate to queueEntry

Commit:
LPS-154071 buildService

Commit:
LPS-154071 Add delete By sentDate to queueEntry

Commit:
LPS-154071 buildService

Commit:
LPS-154071 Add CheckNotificationQueueEntryMessageListener

Kvale1733 committed:
COMMERCE-8534 Add table eligibility entry path

timpak committed:
LPS-152059 Declare the var before usage

ling-alan-huang committed:
LPS-154006 SF

timpak committed:
LPS-153521 Assert the field isn't updated

timpak committed:
LPS-152064 Press the entry key after typing the column name

bryceosterhaus committed:
LPS-153649 - Make code more functional and avoid calling map on each render

ealonso committed:
LPS-154680 SF

22LucaGiannotta22 committed:
COMMERCE-9261 Fix locator and adapt test

ealonso committed:
LPS-154680 Reuse FragmentEntryLinkUtil.getFragmentEntryLinkJSONObject to avoid duplicated code

victorg1991 committed:
LPS-154513 Show correct styles depending on the permissions

ealonso committed:
LPS-154680 Adds new method to accept the defaultFragmentRendererContext since in some cases we want to set other parameters

ealonso committed:
LPS-154680 Uses ContentPageEditorPortletKeys.CONTENT_PAGE_EDITOR_PORTLET since the namespace is always the same

ealonso committed:
LPS-154680 Uses HttpServletRequest and HttpServletResponse

baharturk committed:
COMMERCE-9337 Add CanChangeQuantityOfASubscriptionProductInMiniCart test

victorg1991 committed:
LPS-154513 Not needed

baharturk committed:
COMMERCE-9337 SF

baharturk committed:
COMMERCE-9337 Auto SF

ealonso committed:
LPS-154680 Adds fragmentEntryId

adolfopa committed:
LPS-152800 Include key property in search container

Make the default implementation work with BaseModel (the common case
now). Item selectors for other things will be able to customize that
value.

adolfopa committed:
LPS-152800 Baseline

baharturk committed:
COMMERCE-9337 Updates after review

Kvale1733 committed:
COMMERCE-8975 Move steps to setup and simplify setup

Kvale1733 committed:
COMMERCE-8975 Add UseMappedProductSearchbar test

shuyangzhou committed:
LPS-154700 Most invalid json string is not json string at all, do the cheap shortcut checking before trying to parse it.

SelenaAungst committed:
LPS-154690 Modified sidebar body position.

Kvale1733 committed:
COMMERCE-8534 Modify setEntryEligibility macro to assert entries, navigate to eligibility tabs, and select all eligibility options

Kvale1733 committed:
COMMERCE-8534 Add ACCOUNT_ORDER_DEFAULTS_ENTRY path

Kvale1733 committed:
COMMERCE-8534 Add gotoMiniumAccountOrderDefaults macro

Kvale1733 committed:
COMMERCE-8534 Add AddAndRemoveANewDefaultPaymentMethod test

Kvale1733 committed:
COMMERCE-8535 Add setDefaultAccountPaymentMethod macro

Kvale1733 committed:
COMMERCE-8535 Add default payment method and not eligble payment methods assert to initializersCheckout macro

Kvale1733 committed:
COMMERCE-8535 Add CompleteCheckoutWithDefaultPaymentMethod test

Kvale1733 committed:
COMMERCE-8536 Add CompleteCheckoutWhereDefaultPaymentMethodIsNotPresentIntoOrderType test

Kvale1733 committed:
COMMERCE-8535 Add EditPaymentMethod macro

Kvale1733 committed:
COMMERCE-8537 Add CompleteCheckoutWhereDefaultPaymentMethodIsPresentIntoOrderType test

Kvale1733 committed:
COMMERCE-8538 Add CompleteCheckoutWhereDefaultPaymentMethodIsPresentIntoOrderTypeButDisabledIntoChannel test

Kvale1733 committed:
COMMERCE-8539 Add CompleteCheckoutWhereDefaultPaymentMethodIsDisabledIntoChannel test

Kvale1733 committed:
COMMERCE-8542 Add RemoveOfflinePaymentLinkedAsDefaultPaymentMethod test

Kvale1733 committed:
COMMERCE-8542 Remove duplicate and discarded test stubs

Luizcarlosqueiroz committed:
LPS-154012 Add data-qa-id to subtype

Luizcarlosqueiroz committed:
LPS-154012 Add automated functional tests

Kvale1733 committed:
COMMERCE-8542 Fix tests

Luizcarlosqueiroz committed:
LPS-154012 Js test snapshots update

Luizcarlosqueiroz committed:
LPS-15401 SF

WYZ-Issac committed:
LPS-149484 Add a test to ensure that a user can publish a thread with a tag

ling-alan-huang committed:
LPS-154006 Fix description

ling-alan-huang committed:
LPS-154006 Rename test cases

ling-alan-huang committed:
LPS-154006 Generated

ling-alan-huang committed:
LPS-153991 Auto SF

liuxingye committed:
LPS-154399 Add macro for edit friendly URL

slnn committed:
LPS-154675 Fix NPE, the dlViewFileEntryDisplayContext did not be set when the request dispatched from DLInfoPanelFileEntryOwnerDynamicSection.modify(StringBundler, PageContext)

liuxingye committed:
LPS-154399 Add macros for doing operation in history iframe

liuxingye committed:
LPS-154399 Add macro for go to friendly URL history

liuxingye committed:
LPS-154399 Add paths for friendly URL history

liuxingye committed:
LPS-154399 Add tests for DM friendly URL history

liuxingye committed:
LPS-154399 Add FF references for Poshi tests

jkappler committed:
LPS-154713 Change configuration class accordingly, now that we can set language keys as default values

jkappler committed:
LPS-154713 Build lang

brianchandotcom committed:
artifact:ignore document-library-web 6.0.84 artifact properties

brianchandotcom committed:
artifact:ignore fragment-impl 4.0.37 change log

brianchandotcom committed:
artifact:ignore fragment-impl 4.0.37 artifact properties

brianchandotcom committed:
artifact:ignore fragment-impl 4.0.37 prep next

brianchandotcom committed:
artifact:ignore fragment-web 4.0.38 change log

brianchandotcom committed:
artifact:ignore fragment-web 4.0.38 artifact properties

brianchandotcom committed:
artifact:ignore fragment-web 4.0.38 prep next

brianchandotcom committed:
artifact:ignore journal-web 5.0.76 change log

brianchandotcom committed:
artifact:ignore journal-web 5.0.76 prep next

brianchandotcom committed:
artifact:ignore journal-web 5.0.76 artifact properties

brianchandotcom committed:
artifact:ignore knowledge-base-web 5.0.38 change log

brianchandotcom committed:
artifact:ignore knowledge-base-web 5.0.38 prep next

brianchandotcom committed:
artifact:ignore knowledge-base-web 5.0.38 artifact properties

brianchandotcom committed:
artifact:ignore layout-admin-web 5.0.77 change log

brianchandotcom committed:
artifact:ignore layout-admin-web 5.0.77 artifact properties

brianchandotcom committed:
artifact:ignore layout-admin-web 5.0.77 prep next

brianchandotcom committed:
artifact:ignore layout-content-page-editor-web 3.0.96 change log

brianchandotcom committed:
artifact:ignore layout-content-page-editor-web 3.0.96 artifact properties

brianchandotcom committed:
artifact:ignore layout-content-page-editor-web 3.0.96 prep next

brianchandotcom committed:
artifact:ignore object-admin-rest-impl 1.0.38 change log

liuxingye committed:
LPS-149484 Add path for thread tag

brianchandotcom committed:
artifact:ignore object-admin-rest-impl 1.0.38 artifact properties

brianchandotcom committed:
artifact:ignore object-admin-rest-impl 1.0.38 prep next

brianchandotcom committed:
artifact:ignore object-rest-impl 1.0.38 change log

brianchandotcom committed:
artifact:ignore object-rest-impl 1.0.38 prep next

brianchandotcom committed:
artifact:ignore object-rest-impl 1.0.38 artifact properties

brianchandotcom committed:
artifact:ignore object-web 1.0.72 change log

liuxingye committed:
LPS-149484 Update macros for select and view tag

brianchandotcom committed:
artifact:ignore object-web 1.0.72 artifact properties

brianchandotcom committed:
artifact:ignore object-web 1.0.72 prep next

brianchandotcom committed:
artifact:ignore portal-search-web 6.0.47 change log

liuxingye committed:
LPS-149484 Update macro for skip unnecessary step

brianchandotcom committed:
artifact:ignore portal-search-web 6.0.47 artifact properties

brianchandotcom committed:
artifact:ignore portal-search-web 6.0.47 prep next

brianchandotcom committed:
artifact:ignore questions-web 2.0.48 change log

liuxingye committed:
LPS-149484 Update test for select and view tag

brianchandotcom committed:
artifact:ignore site-navigation-menu-item-layout 4.0.18 change log

brianchandotcom committed:
artifact:ignore site-navigation-menu-item-layout 4.0.18 prep next

brianchandotcom committed:
artifact:ignore site-navigation-menu-item-layout 4.0.18 artifact properties

brianchandotcom committed:
site-navigation-menu-item-layout 4.0.18 apply

brianchandotcom committed:
artifact:ignore site-initializer-extender 1.0.39 change log

brianchandotcom committed:
artifact:ignore site-initializer-extender 1.0.39 prep next

brianchandotcom committed:
artifact:ignore site-initializer-extender 1.0.39 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-admin-catalog-impl 4.0.45 change log

lfbesada committed:
LPS-152836 Change AssetVocabularyItemSelectorViewDescriptor default displayStyle to list (Table in UI)

brianchandotcom committed:
artifact:ignore headless-commerce-admin-catalog-impl 4.0.45 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-admin-catalog-impl 4.0.45 artifact properties

brianchandotcom committed:
artifact:ignore headless-commerce-admin-site-setting-impl 4.0.26 change log

brianchandotcom committed:
artifact:ignore headless-commerce-admin-site-setting-impl 4.0.26 prep next

brianchandotcom committed:
artifact:ignore headless-commerce-admin-site-setting-impl 4.0.26 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore commerce-product-tax-category-web 4.0.17 change log

brianchandotcom committed:
artifact:ignore commerce-product-tax-category-web 4.0.17 prep next

brianchandotcom committed:
artifact:ignore commerce-product-tax-category-web 4.0.17 artifact properties

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-test-util 12.0.0 change log

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-test-util 12.0.0 prep next

brianchandotcom committed:
artifact:ignore dynamic-data-mapping-test-util 12.0.0 artifact properties

brianchandotcom committed:
artifact:ignore questions-web 2.0.49 artifact properties

brianchandotcom committed:
artifact:ignore questions-web 2.0.49 prep next

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore analytics-reports-web 2.0.47 change log

brianchandotcom committed:
artifact:ignore analytics-reports-web 2.0.47 artifact properties

brianchandotcom committed:
artifact:ignore analytics-reports-web 2.0.47 prep next

brianchandotcom committed:
artifact:ignore frontend-theme-dxp-override 4.0.24 prep next

brianchandotcom committed:
artifact:ignore frontend-theme-dxp-override 4.0.24 artifact properties

brianchandotcom committed:
artifact:ignore portal-dao-db 5.0.16 change log

brianchandotcom committed:
artifact:ignore portal-dao-db 5.0.16 prep next

brianchandotcom committed:
artifact:ignore portal-dao-db 5.0.16 artifact properties

brianchandotcom committed:
artifact:ignore portal-reports-engine-console-jasper 4.0.15 change log

brianchandotcom committed:
artifact:ignore portal-reports-engine-console-jasper 4.0.15 prep next

brianchandotcom committed:
artifact:ignore portal-reports-engine-console-jasper 4.0.15 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-designer-web 5.0.58 change log

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-designer-web 5.0.58 prep next

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-designer-web 5.0.58 artifact properties

brianchandotcom committed:
artifact:ignore saml-persistence-service 5.0.31 change log

brianchandotcom committed:
artifact:ignore saml-persistence-service 5.0.31 prep next

brianchandotcom committed:
artifact:ignore saml-persistence-service 5.0.31 artifact properties

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.1 change log

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.1 prep next

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.1 artifact properties

brianchandotcom committed:
artifact:ignore segments-asah-connector 3.0.19 change log

brianchandotcom committed:
artifact:ignore segments-asah-connector 3.0.19 prep next

brianchandotcom committed:
artifact:ignore segments-asah-connector 3.0.19 artifact properties

brianchandotcom committed:
artifact:ignore sharepoint-soap-repository 5.0.8 change log

brianchandotcom committed:
artifact:ignore sharepoint-soap-repository 5.0.8 artifact properties

brianchandotcom committed:
artifact:ignore sharepoint-soap-repository 5.0.8 prep next

brianchandotcom committed:
build.gradle auto SF

Commit:
LRQA-75223 Add step to assert the alert message displays

jkappler committed:
LPS-147440 Move code to SiteInitializerUtil and also add a whitelist of those properties that we accept

Hijky committed:
LPS-153229 Add a macro to sort web content with priority

Hijky committed:
LPS-153229 Add a test to sort web content with priority

lfbesada committed:
LPS-154601 Prevents NullPointerException

thaiscabrals committed:
LPS-154747 Fix Tests for TestrayProjectsTable

thaiscabrals committed:
LPS-154747 SF

brianchandotcom committed:
LPS-154675 sort

Kvale1733 committed:
COMMERCE-8975 SF

brianchandotcom committed:
LPS-151362 Sort

brianchandotcom committed:
LPS-151362 Wordsmith

brianchandotcom committed:
LPS-151362 regen

brianchandotcom committed:
LPS-153649 Remove

brianchandotcom committed:
LPS-153649 Rename

brianchandotcom committed:
LPS-153649 auto SF

brianchandotcom committed:
LPS-147440 rename

brianchandotcom committed:
LPS-147440 private

brianchandotcom committed:
LPS-147440 auto SF

brianchandotcom committed:
LPS-147440 Rename

Commit:
LPS-154071 Add NotificationTypeRegistry

Commit:
LPS-154071 Add notifictionType To ObjectDefinitionDeployerImpl

Commit:
LPS-154071 Move and fix tests

Commit:
LPS-154071 Format to field

Commit:
LPS-154071 SF
